common/persistence/elasticsearch/elasticsearchVisibility.go

```diff
@@ -318,18 +318,20 @@ func (v *esVisibilityManager) getSearchResult(request *p.ListWorkflowExecutionsR
 }
 
 func (v *esVisibilityManager) getListWorkflowExecutionsResponse(searchHits *elastic.SearchHits,
-	token *esVisibilityPageToken, isOpen bool) (*p.ListWorkflowExecutionsResponse, error) {
+	token *esVisibilityPageToken, isOpen bool, pageSize int) (*p.ListWorkflowExecutionsResponse, error) {
 
 	response := &p.ListWorkflowExecutionsResponse{}
 	actualHits := searchHits.Hits
 	numOfActualHits := len(actualHits)
 
-	nextPageToken, err := v.serializePageToken(&esVisibilityPageToken{From: token.From + numOfActualHits})
-	if err != nil {
-		return nil, err
+	if numOfActualHits == pageSize {
```

Comment:
In my understanding, when numOfActualHits<=pageSize, the nextPageToken should be empty. Isn't it? I think I miss some knowledge background here, can you leave some comment to explain?

Reply:
you understanding is correct. when numOfActualHits<=pageSize, the nextPageToken should be empty